app/src/main/assets/about.html

```diff
@@ -1,12 +1,24 @@
 <p>
     MusicBrainz is an open music encyclopedia that collects music metadata and makes it available to the public.
+    <br><br>
```

Comment:
Lets both the points from https://musicbrainz.org/ verbatim and since adding `a` href links is not simple also add a line like `Visit https://musicbrainz.org/ to know more`.

Reply:
Done